__tests__/src/selectors/manifests.test.js

```diff
@@ -59,6 +62,14 @@ describe('getManifestoInstance', () => {
     const received = getManifestoInstance(state, { manifestId: 'x' });
     expect(received.id).toEqual('http://iiif.io/api/presentation/2.1/example/fixtures/19/manifest.json');
   });
+  it('creates a manifesto collection instance and patches getSequences', () => {
```

Comment:
Not important, but could you insert an empty line above? The eslint rules actually don't enforce that but it would be more consistent. Thanks!